src/main/java/no/digipost/http/client/DigipostHttpClientDefaults.java

```diff
@@ -128,7 +127,9 @@
         public final int maxTotal;
         public final int maxPerRoute;
 
-        private ConnectionAmount(int maxTotal, int maxPerRoute) {
+        public ConnectionAmount(int maxTotal, int maxPerRoute) {
```

Comment:
Synes `ConnectionAmount` bør følge samme pattern som `DigipostHttpClientMillisecondTimeouts`. Altså bør `ConnectionAmount` extractes som egen klasse -> `DigipostHttpClientConnectionAmount`, og ha litt metoder for å endre verdiene. `DigipostHttpClientMillisecondTimeouts` har også en metode `isPotentiallyDangerous`, som kalles i `DigipostHttpClientFactory`. Bør `DigipostHttpClientConnectionAmount` også ha en slik? Er det noen verdier som bør være gyldige, men som kun bør brukes f.eks. test?

Jeg foreslår at vi parprogrammerer disse endringene :)

@froden nevnte at han synes det var litt forvirrende at for å lage custom `DigipostHttpClientMillisecondTimeouts` måtte han begynne med `DigipostHttpClientDefaults.DEFAULT_TIMEOUTS_MS` og "modifisere" denne, men jeg foreslår å fikse dette i en egen pull-request, og at vi nå følger "patternet" som allerede er etablert her.